.github/workflows/build.yml

```diff
@@ -30,9 +30,8 @@ jobs:
       - name: Setup Graalvm
         uses: graalvm/setup-graalvm@v1
         with:
-          version: '21.3.0'
-          java-version: '11'
-          components: 'native-image'
+          java-version: '17.0.8'
+          distribution: 'graalvm'
```

Comment:
While desiderable, I think Java update should go into a separate PR, unless there's an explicit dependency with this version

Reply:
Graalvm for java 11 is throwing an error while building on gh action with the [following error](https://github.com/oracle/graal/issues/3144): I was checking an the only solution I found was to upgrade the java version

Reply:
Fair enough